### PR TITLE
BootstrapMinimal job update for recreating cert secrets

### PR DESCRIPTION
**Pull request by ddevadat**
This is done by the job Deploy /Kubernetes /BootstrapMinimal
But this job is not idempotent and fails when trying to re run task: Creating docker secrets

This was because if it tries to reexecutes creation of existing imagepull secrets..it fails and hence the execution for further steps.

added a flag to ignore errors to continue with the play

https://github.com/AmitKaushik-Oracle/sunbird-devops/blob/177d105a1349f9a7ce96f6ff95524f5031e4b8c1/kubernetes/ansible/bootstrap_minimal.yaml

### CODE REVIEW

**Comment by ddevadat** [comment]
merged wrk-ard-bootstrap to fixoke

**Comment by heungheung** [comment]
I have doubt 

This is a direct modification to the file
   kubernetes/ansible/bootstrap_minimal.yaml
That means we CANNOT keep the original sunbird-project repo intact. 

This is NOT a desired way to handle things as sunbird-project might not be able to directly uptake our change(s).

Is it possible, instead of making a direct chanage, we add new file?

@ddevadat shall we revert this PR?

File: kubernetes/ansible/bootstrap_minimal.yaml [hunk 1]
```diff
@@ -34,6 +34,8 @@
       when: imagepullsecrets|length > 0
       with_items:
         - "{{ bootstrap_namespace.split(',') }}"
+      ignore_errors: yes
```
Comment (heungheung) [inline kubernetes/ansible/bootstrap_minimal.yaml:37]: I will highly recommend we do NOT make direct changes to the sunbird-project file

Comment (ddevadat) [reply kubernetes/ansible/bootstrap_minimal.yaml:37]: we can create an oci specific pipeline ( Jenkinsfile.oci ) ---> which will call then oci specific ansible play ( bootstrap_minimal_oci.yaml ) 
bootstrap_minimal_oci.yaml ---> will have the fix

Will that approach work?

Comment (ddevadat) [reply kubernetes/ansible/bootstrap_minimal.yaml:37]: reverted the PR. Will add a new file

**Pull request merged**